src/lib/layout/mobile/NavDrawer.tsx

```diff
@@ -29,94 +30,150 @@ import { CustomIcon } from "lib/components/icon";
 import { useIsCurrentPage } from "lib/hooks";
 import { usePublicProjectStore } from "lib/providers/store";
 
+// eslint-disable-next-line sonarjs/cognitive-complexity
```

Comment:
I think if we separate a navbar objects, it should be fine. Can you test it. Thank!